### PR TITLE
Add modder and nominator counts to beatmap cards

### LINKED ISSUE #11: Indicate amount of modders/potential nominators on the beatmap tile
That way one could immediately see which maps are already highly contested without having to click on the burger menu every time:
grafik

The colors should match the colors used in the burger menu.
Especially for BNs this should be very useful as maps with 2+ nominators probably don't need more than that.

### PR DESCRIPTION
**Pull request by cl8n**
resolves #11

maybe the laziest design ever but this is fine

### CODE REVIEW

**Comment by venix12** [comment]
those should have tooltips

File: resources/sass/app.scss [hunk 1]
```diff
@@ -225,14 +225,28 @@ hr {
...
     padding-right: 1em;
+    box-shadow: 0px 0px 15px rgb(37, 37, 37);
+
+    &--modders {
+        background-color: $lightblue;
+        color: $darkgray;
+    }
+
+    &--nominators {
+        background-color: #deaae2;
```
Comment (venix12) [inline resources/sass/app.scss]: you could make it into $pink variable :eyes:

Comment (cl8n) [reply resources/sass/app.scss]: same with half of the other rules but I'm not down 2 refactor whole css in this pr

Comment (venix12) [reply resources/sass/app.scss]: ok i will make it myself then ¯\_(ツ)_/¯

**Pull request merged**